force-app/main/default/labels/CustomLabels.labels-meta.xml

```diff
@@ -236,6 +243,20 @@
         <shortDescription>Missing SObject Type</shortDescription>
         <value>Your query must include an object.</value>
     </labels>
+    <labels>
+        <fullName>Monthly_On_Day</fullName>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Monthly on Day</shortDescription>
+        <value>Monthly on Day {0}</value>
+    </labels>
+    <labels>
+        <fullName>Monthly_On_The</fullName>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Monthly On The</shortDescription>
+        <value>Monthly on the {0} {1}</value>
+    </labels>
```

Comment:
Do we still need this one since we ended up using picklist values?

Reply:
Yeah we need this since we are still construction the picklist options on the fly